amqp1jobs/config.go

```
@@ -0,0 +1,197 @@
+	Key      string          `mapstructure:"key"`
+	Cert     string          `mapstructure:"cert"`
+	AuthType ClientAuthType  `mapstructure:"client_auth_type"`
+	// auth type internal
+	auth tls.ClientAuthType
+}
+
+func (c *config) InitDefault() error {
+	const op = errors.Op("amqp1_init_default")
+	// all options should be in sync with the pipeline defaults in the ConsumerFromPipeline method
+	if c.ExchangeType == "" {
+		c.ExchangeType = "direct"
```

golangci-lint (1.64.8)

string `direct` has 2 occurrences, make it a constant

(goconst)